client/my-sites/checkout/concierge-session-nudge/index.jsx

```diff
@@ -43,12 +43,12 @@ export class ConciergeSessionNudge extends React.Component {
 	};
 
 	render() {
-		const { selectedSiteId, isLoading, hasProductsList, hasSitePlans } = this.props;
-		const title = 'Checkout ‹ Expert Session';
+		const { selectedSiteId, isLoading, hasProductsList, hasSitePlans, translate } = this.props;
+		const title = translate( 'Checkout ‹ Support Session' );
```

Comment:
It's not required, but might be nice to add [a comment](https://github.com/Automattic/i18n-calypso#more-translate-examples) for the translators here, since "Checkout" is a hyphen away from being a noun or a verb and the use-case is a little ambiguous without the code/PR.

Reply:
thanks - I added a comment